Dockerfile.windows

```diff
@@ -171,70 +171,28 @@ ENV GO_VERSION=1.7.3 `
     FROM_DOCKERFILE=1
 
 RUN `
-  $ErrorActionPreference = 'Stop'; `
-  $ProgressPreference = 'SilentlyContinue'; `
-  `
-  Function Test-Nano() { `
-    $EditionId = (Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion' -Name 'EditionID').EditionId; `
-    return (($EditionId -eq 'ServerStandardNano') -or ($EditionId -eq 'ServerDataCenterNano') -or ($EditionId -eq 'NanoServer')); `
-  }`
-  `
-  Function Download-File([string] $source, [string] $target) { `
```

Comment:
To be clear I'm saying to make this work on internal builds which aren't public, this is necessary. I cannot comment on what will eventually be released. But I need it to keep running internally now. Further, with some code I'm working on, there will hopefully be a single unified dockerfile for both nano and server core.